Library/Formula/sassc.rb

```diff
@@ -2,8 +2,8 @@
 
 class Sassc < Formula
   homepage "https://github.com/sass/sassc"
-  url "https://github.com/sass/sassc/archive/3.1.0.tar.gz"
-  sha1 "573224ad922b46ea1a568807ddcbc7de41a4254d"
+  url "https://github.com/sass/sassc/archive/3.2.0.tar.gz"
+  sha1 "73454e810aba782466df4dcf59e73142bece37f494776d8bab2e3dae88992a24"
```

Comment:
You need to change the wording as well. Still reads `sha1`. Otherwise, perfect, thanks!